types/scte224v20200407/sctestructures.go

```diff
@@ -0,0 +1,325 @@
+package scte224v20200407
+
+import (
+	"encoding/xml"
+	"log"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Comcast/scte224structs/types/scte224v20180501/adi30"
+)
+
+const schemaLocation = "https://www.scte.org/standards-development/library/standards-catalog/ansiscte-224-2018r1/"
+
+// Structs for SCTE 224 2018 ESNI Objects.
```

Comment:
is this just copies over from 2018 then?

Reply:
Oops yes, will update